docs/known-issues.md

```diff
@@ -13,3 +13,5 @@ Known issue = any major user-facing bug or lack of user-facing feature that:
 | [Writer [Require] fields]({{urlRoot}}/content/gameobject/reading-and-writing-component-data.md#how-to-update-spatialos-component-properties): When you use `Writer.Send` to update a component's properties, it causes the SpatialOS GDK to update all properties of the component, even if you have set it to only update some of the properties. | 2018/09/25 | None         | None                                                                  | No     |
 | [Reader [Require] fields]({{urlRoot}}/content/gameobject/reading-and-writing-component-data#how-to-react-to-component-property-changes): The `ISpatialComponentUpdate update` argument of `Reader.ComponentUpdated` may indicate that a component property has changed even when it has not.                                          | 2018/09/25 | None         | Verify manually whether a component property has actually changed.    | No     |
 | [Reader [Require] fields]({{urlRoot}}/content/gameobject/reading-and-writing-component-data.md#how-to-react-to-component-property-changes): `Reader.<component property name>Updated` may be invoked even if `<component property name>` did not change.                                                                                 | 2018/09/25 | None         | Verify manually whether a component property has actually changed.    | No     |
+| There is a bug in the preview version of the [High Definition Render Pipeline](https://blogs.unity3d.com/2018/03/16/the-high-definition-render-pipeline-focused-on-visual-quality/), where shaders do not fully compile and appear visually darker than intended. | 2018/10/12 | [566245](https://forum.unity.com/threads/shadergraph-subgraph-importing-bug-on-shared-project.566245/)         | Manually reimport shaders.                                                                  | No     |
```

Comment:
Suggest:
There is a bug in the preview version of the [High Definition Render Pipeline (Unity blog)](https://blogs.unity3d.com/2018/03/16/the-high-definition-render-pipeline-focused-on-visual-quality/)

{{ We want to have a better way to tell people they are leaving our docs when they hit certain links, but, for now, we are going with where they are headed in brackets.